typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -568,25 +569,58 @@ export abstract class HyperlaneAppGovernor<
         );
         this.canPropose[chain].set(safeAddress, canPropose);
       } catch (error) {
+        if (!(error instanceof Error)) {
```

Comment:
When would this happen?

Reply:
I'm not sure, I mainly did it to keep the typings further down - should I instead just assert?

```ts
} catch (error) {
	assert(error instanceof Error, 'Error is not an instance of Error');
```

Reply:
Yeah that could work. does `catch (error: Error)` work? Otherwise an `as Error` seems also fine here

Reply:
`error: Error` won't work because it expects `any` or `unknown` - i've changed it to `as Error` when reading the error message